CONTRIBUTING.md

```diff
@@ -25,3 +25,23 @@ Kick off your TDD (i.e. goconvey powered) development environment as follows:
 ```
 
 Visit localhost:8080 to see the test results in real time.
+
+## Releasing
+
+This project follows the standard [Conjur community release guidelines](https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#release-process).
+
+In particular, for each release you should:
+
+- Review the changes since the last release, and update the [version](./VERSION)
+  following [semantic versioning](https://semver.org/).
+- Determine whether any dependencies were added, removed, or updated in
+  [`go.mod`](./go.mod) since the last release. If any changes have been made,
+  update the [NOTICES](./NOTICES.txt) file.
+- Update the [CHANGELOG](./CHANGELOG.md) to reflect the new version.
+- Commit the changes to the files above in a branch and submit a version bump PR
+- Once the PR has been merged, tag the version using
```

Comment:
For tagging and creating a release, do we want to add a reference to the community guidelines?:
https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#tagging
https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#github-releases

Reply:
I link to the overall release process above, which links to both of these